cpp/src/parquet/reader_test.cc

```diff
@@ -116,6 +116,10 @@ std::string rle_dict_uncompressed_corrupt_checksum() {
   return data_file("rle-dict-uncompressed-corrupt-checksum.parquet");
 }
 
+std::string concatenated_gzip_members() {
+  return data_file("concatenated_gzip_members.parquet");
```

Comment:
This requires to upload the file to apache/parquet-testing repo first, then sync the submodule directory in the apache/arrow in this PR.

Reply:
If is gurantee that we can read this. we'd better to upload this file to `parquet-testing` repo and make other parquet impl able to testing it

Reply:
https://github.com/apache/parquet-testing/pull/41
do you have access writes? I opened a pull request fort hat